.drone.star

```diff
@@ -89,6 +89,7 @@ config = {
 			},
 			'runningOnOCIS': True,
 			'filterTags': 'not @skip and not @skipOnOCIS',
+			'screenShots': True,
```

Comment:
debug or do we need the screenshots for longterm?

Reply:
I think we can leave it for now, will be helpful as soon as tests fail in ocis.